apps/system/js/utility_tray.js

```diff
@@ -614,10 +701,33 @@ window.UtilityTray = {
     }
   },
 
-  _shouldPrevent: function ut_shouldPrevent(currentTarget) {
-    var targeted = [this.grippy, this.statusbarIcons, this.topPanel];
-    return targeted.some(function(target) {
-      return target === currentTarget;
-    });
+  _shouldPrevent: function ut_shouldPrevent(target) {
+    return [
+      this.statusbarIcons,
```

Comment:
I'm happy to remove this. There were no comments explaining why this was required, so I kept it to be on the safe side. But let's strip as much old code as possible :)